docs/server/source/simple-network-setup.md

```diff
@@ -0,0 +1,209 @@
+
+A **BigchainDB Network** (*Network* henceforth) is a set of **4 or more BigchainDB Nodes** (*Node* henceforth). Every Node is independently managed by a Member, and runs an instance of the [BigchainDB Server software][bdb:software]. At the **Genesis** of a Network, there **MUST** be at least **4** Nodes ready to connect. After the Genesis, a Network can dynamically add new Nodes or remove old Nodes.
+
+A Network will stop working if more than one third of the Nodes are down. The bigger a Network, the more failures it can handle. A Network of size 4 can tolerate only 1 failure, so if 3 out of 4 Nodes are online, everything will work as expected. Eventually, the Node that was offline will automatically sync with the others.
+
+
+## Before we start
+This tutorial assumes you have basic knowledge on how to manage a GNU/Linux machine. The commands are tailored for an up-to-date *Debian-like* distribution (we use an **Ubuntu 17.10** Virtual Machine on Microsoft Azure): if you are on a different distribution you might need to adapt the name of the packages installed.
+
+We don't make any assumptions on **where** you run the Node.
+You can run the BigchainDB Server on a Virtual Machine on the cloud, on a machine in your data center, on a Raspberry Pi. Just make sure that your Node is reachable by the other Nodes, a **non-exhaustive list of examples**:
+- **good**: all Nodes running in the cloud using public IPs.
+- **bad**: some Nodes running in the cloud using public IPs, some Nodes in a private network.
+- **good**: all Nodes running in a private network.
+
+The rule of thumb is: if Nodes can ping each other, then you are good to go.
+
+The next sections are labelled with **Member** or **Coordinator**, depending on who should follow the instructions. Remember, a Coordinator is also a Member.
+
+## Member: Set up a Node
+Every Member in the Network **must** set up its own Node. The process consists of installing three components, BigchainDB Server, Tendermint Core, and MongoDB, and configure the firewall.
+
+**Important note on security: it's up to the Member to harden their system.**
+
+### Install the required software
+Make sure your system is up to date.
+
+```
+sudo apt update
+sudo apt full-upgrade
+```
+
+#### Install BigchainDB Server
+BigchainDB Server requires **Python 3.6+**, make sure your system has it. Install the required packages:
+
+```
+sudo apt install -y python3-pip libssl-dev
+```
+
+Now install the latest version of BigchainDB. Check the [project page on PyPI][bdb:pypi] for the last version (`2.0.0a5` at the time of writing) and install it:
+
+```
+sudo pip3 install bigchaindb==2.0.0a5
```

Comment:
Let's change this to 2.0.0a6 and maybe not mention any version number in the line above